phoenix-pig/src/test/java/org/apache/phoenix/pig/util/TypeUtilTest.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.phoenix.pig.util;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.util.List;
+
+import org.apache.phoenix.pig.writable.PhoenixPigDBWritable;
+import org.apache.pig.ResourceSchema.ResourceFieldSchema;
+import org.apache.pig.data.DataType;
+import org.apache.pig.data.Tuple;
+import org.junit.Test;
+
+import com.google.common.collect.Lists;
+
+public class TypeUtilTest {
+
+    @Test
+    public void testTransformToTuple() throws Exception {
+        PhoenixPigDBWritable record = mock(PhoenixPigDBWritable.class);
+        List<Object> values = Lists.newArrayList();
+        values.add("213123");
+        values.add(1231123);
+        values.add(31231231232131L);
+        values.add("bytearray".getBytes());
+        when(record.getValues()).thenReturn(values);
+
+        ResourceFieldSchema field = new ResourceFieldSchema().setType(DataType.CHARARRAY);
```

Comment:
I added a few for these changes as I noticed no tests existed around this whatsoever. Can we get this in, and may be open a JIRA for adding tests around other data types?